libs/akita-ng-entity-service/src/lib/ng-entity-service.spec.ts

```diff
@@ -277,7 +277,7 @@ describe('NgEntityService', () => {
           TestServiceWithInlineConfig,
           {
             provide: NG_ENTITY_SERVICE_CONFIG,
-            useValue: {} as NgEntityServiceGlobalConfig,
+            useValue: { replayLoader: true } as NgEntityServiceGlobalConfig,
```

Comment:
I set the replayLoader to true for all 'get' tests. I assume this isn't what we want but I only placed replayLoader on the global config. 
- Should I add replayLoader to service level configs as well?
- Or just modify the global config for my single test?